app/components/provider_interface/user_org_permission_breakdown_component.rb

```diff
@@ -0,0 +1,46 @@
+module ProviderInterface
+  class UserOrgPermissionBreakdownComponent < ViewComponent::Base
+    attr_reader :provider, :permission
+
+    def initialize(provider:, permission:)
+      @provider = provider
+      @permission = permission
+    end
+
+    def permitted_by_providers
+      @permitted_by_providers ||= Provider.where(id: training_providers_with_permissions.pluck(:id))
+                                          .or(Provider.where(id: ratifying_providers_with_permissions.pluck(:id)))
+    end
+
+    def prohibited_by_providers
+      @prohibited_by_providers ||= Provider.where(id: training_providers_without_permissions.pluck(:id))
+                                           .or(Provider.where(id: ratifying_providers_without_permissions.pluck(:id)))
+    end
+
+  private
+
+    def training_providers_with_permissions
+      Provider.joins(:training_provider_permissions)
+              .where(provider_relationship_permissions: { ratifying_provider: provider,
+                                                          "training_provider_can_#{permission}" => true })
+    end
+
+    def ratifying_providers_with_permissions
+      Provider.joins(:ratifying_provider_permissions)
+              .where(provider_relationship_permissions: { training_provider: provider,
+                                                          "ratifying_provider_can_#{permission}" => true })
+    end
+
+    def training_providers_without_permissions
+      Provider.joins(:training_provider_permissions)
+              .where(provider_relationship_permissions: { ratifying_provider: provider,
+                                                          "training_provider_can_#{permission}" => false })
+    end
+
+    def ratifying_providers_without_permissions
+      Provider.joins(:ratifying_provider_permissions)
+              .where(provider_relationship_permissions: { training_provider: provider,
+                                                          "ratifying_provider_can_#{permission}" => false })
+    end
```

Comment:
I think these will need to be the other way round - when the given Provider is the ratifying provider in the relationship, then we will check the `ratifying_provider_can_#{permissions}` for that relationship.